src/mlpack/tests/main_tests/radical_test.cpp

```diff
@@ -0,0 +1,117 @@
+/**
+ * @file radical_test.cpp
+ * @author Manish Kumar
+ *
+ * Test mlpackMain() of radical_main.cpp.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#include <string>
+
+#define BINDING_TYPE BINDING_TYPE_TEST
+static const std::string testName = "Radical";
+
+#include <mlpack/core.hpp>
+#include <mlpack/core/util/mlpack_main.hpp>
+#include "test_helper.hpp"
+#include <mlpack/methods/radical/radical_main.cpp>
+
+#include <boost/test/unit_test.hpp>
+#include "../test_tools.hpp"
+
+using namespace mlpack;
+
+struct RadicalTestFixture
+{
+ public:
+  RadicalTestFixture()
+  {
+    // Cache in the options for this program.
+    CLI::RestoreSettings(testName);
+  }
+
+  ~RadicalTestFixture()
+  {
+    // Clear the settings.
+    bindings::tests::CleanMemory();
+    CLI::ClearSettings();
+  }
+};
+
+BOOST_FIXTURE_TEST_SUITE(RadicalMainTest, RadicalTestFixture);
+
+/**
+ * Check that output Y and W matrix have valid dimensions.
+ */
+BOOST_AUTO_TEST_CASE(RadicalOutputDimensionTest)
+{
+  arma::mat input = arma::randu<arma::mat>(5, 3);
+
+  SetInputParam("input", std::move(input));
+
+  mlpackMain();
+
+  // Check dimension of Y matrix.
+  BOOST_REQUIRE_EQUAL(CLI::GetParam<arma::mat>("output_ic").n_rows,5);
```

Comment:
This is a really minor style issue, can you make sure there is a space between arguments here?  i.e. `n_rows, 5` not `n_rows,5`.  :)